### ISSUE TITLE
Language id differs from vscode default

### ISSUE THREAD

**Issue by mayrholu** [tht13/RST-vscode#19]
Hi @tht13 , 
I really like the extension. It is simple, requires no setup and it works. However it would be nice if the language id would be the default vscode setting "restructuredtext" instead of RST. Right now one has to switch between those two ids two for using the preview or the linter. Would it be possible for you to update the extension at some point? Thanks!

**Comment by floweronthemoon** [comment]
> Hi @tht13 , I really like the extension. It is simple, requires no setup and it works. However it would be nice if the language id would be the default vscode setting "restructuredtext" instead of RST. Right now one has to switch between those two ids two for using the preview or the linter. Would it be possible for you to update the extension at some point? Thanks!

Yes, same issue! @tht13 please add support for "reStructuredText" language mode in VS Code.
This is a neat little extension that is very handy.
